### PR TITLE
Update description text of tentacle, match reality

### PR DESCRIPTION
**Pull request by nok-ko**
## What Does This PR Do
Changes the description of the changeling tentacle ability.

Old:
> [...] and Harm will stun it, and stab it if we are also holding a sharp weapon. Cannot be used while in lesser form.

New:
> [...] and Harm will drag it closer, and stab it if we are also holding a sharp weapon. Cannot be used while in lesser form.

## Why It's Good For The Game
Removes misinformation about a nonexistent stun. Misinformation? Bad.

## Changelog
:cl: Nokko
del: Removed misleading changeling tentacle description that said it could stun.
/:cl:

### CODE REVIEW

File: code/game/gamemodes/changeling/powers/mutations.dm [hunk 1]
```diff
@@ -173,8 +173,8 @@
 	name = "Tentacle"
 	desc = "We ready a tentacle to grab items or victims with. Costs 10 chemicals."
 	helptext = "We can use it once to retrieve a distant item. If used on living creatures, the effect depends on the intent: \
-	Help will simply drag them closer, Disarm will grab whatever they are holding instead of them, Grab will put the victim in our hold after catching it, \
-	and Harm will stun it, and stab it if we are also holding a sharp weapon. Cannot be used while in lesser form."
+	Help will simply drag it closer, Disarm will grab whatever the creature is holding, Grab will put the victim in our hold after catching it, \
```
Comment (S34NW) [inline code/game/gamemodes/changeling/powers/mutations.dm:176]:
```suggestion
	Help will drag the target closer. \
	Disarm will grab whatever the target is holding. \
	Grab will put the target in our grip after we catch it. \
```

File: code/game/gamemodes/changeling/powers/mutations.dm [hunk 2]
```diff
@@ -173,8 +173,8 @@
...
+	and Harm will drag it closer, and stab it if we are also holding a sharp weapon. Cannot be used while in lesser form."
```
Comment (S34NW) [inline code/game/gamemodes/changeling/powers/mutations.dm:177]:
```suggestion
	Harm will drag the target closer and stab it. \
	Cannot be used while in our lesser form."
```

**Review by S34NW — CHANGES_REQUESTED** [review]
Good change, this text was quite old so I'm glad you are giving it some love!

Small suggestions for readability below:
Perhaps you could split each intent onto a new line to make it easier to read, and use "target" instead of "it", as you can grab both living beings or items.

**Review by Kyep — APPROVED** [review]

**Review by AffectedArc07 — CHANGES_REQUESTED** [review]
Please resolve outstanding review

**Review by dearmochi — APPROVED** [review]

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**